pkgs/development/python-modules/django-signal-webhooks/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  django,
+  asgiref,
+  httpx,
+  cryptography,
+  django-settings-holder,
+  pytestCheckHook,
+  pytest-django,
+  freezegun,
+  djangorestframework,
+}:
+
+buildPythonPackage rec {
+  pname = "django-signal-webhooks";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "MrThearMan";
+    repo = "django-signal-webhooks";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Ag2cCKG9IayKh2bHv/bS0na/c6FgyDnQcOwUc6TWr9A=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = [
+    asgiref
+    cryptography
+    django
+    django-settings-holder
+    httpx
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-django
+    freezegun
+    djangorestframework
+  ];
+
+  disabledTests = [
+    # Require network access
+    "test_webhook__single_webhook__sending_timeout"
+    "test_webhook__multiple_webhooks__sending_timeout"
+    "test_webhook_e2e__single_webhook__timeout"
+    "test_webhook_e2e__single_webhook__ngrok"
+    "test_webhook_api__exists"
+    # TypeErorr: 'str' object is not callable
+    "test_webhook__custom_setup"
+  ];
+
+  # Cannot do pythonImportsCheck as it requires Django settings to be configured.
```

Comment:
Shouldn't `export DJANGO_SETTINGS_MODULE=tests.project.settings` in `preCheck` do just that? Why isn't it required for the other tests?

Reply:
It's required for the other tests, but pythonImportsCheck is a different code path that doesn't go through pytest-django(which starts django first, which is needed)

Reply:
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/hooks/python-imports-check-hook.sh, it's a slightly more complicated `python -c 'import <module name>'` so we don't have django set up as needed in the environment.

Same reason why other packages can't do pythonImportsCheck.